pandas/core/arrays/numpy_.py

```diff
@@ -162,6 +162,8 @@ def _from_sequence(cls, scalars, dtype=None, copy=False):
         result = np.asarray(scalars, dtype=dtype)
         if copy and result is scalars:
             result = result.copy()
+        # _coerce_from_sequence_values is useful for StringArray
+        result = cls._coerce_from_sequence_values(result, copy=copy)
```

Comment:
where is this?

Reply:
Sorry, leftover change that I forgot to remove.